{
 "errors": {},
 "symbols": [
  "pytest_faulthandler._cancel_traceback_dump",
  "pytest_faulthandler.pytest_addoption",
  "pytest_faulthandler.pytest_configure",
  "pytest_faulthandler.pytest_enter_pdb",
  "pytest_faulthandler.pytest_exception_interact",
  "pytest_faulthandler.pytest_runtest_protocol",
  "pytest_faulthandler.timeout_support_available"
 ]
}